pkg/http/api.go

```diff
@@ -233,23 +233,26 @@ func (a *api) MarkStatusAsOutboundReady() {
 	a.outboundReadyStatus = true
 }
 
+// Workflow Component: Component specified in yaml (temporal, etc..)
+// Workflow Type: Name of the workflow to run (function name)
+// Instance ID: Identifier of the specific run
 func (a *api) constructWorkflowEndpoints() []Endpoint {
 	return []Endpoint{
 		{
 			Methods: []string{fasthttp.MethodGet},
-			Route:   "workflows/{workflowId}/{instanceId}",
+			Route:   "workflows/{workflowComponent}{workflowType}/{instanceId}",
 			Version: apiVersionV1alpha1,
 			Handler: a.onGetWorkflow,
 		},
 		{
 			Methods: []string{fasthttp.MethodPost},
-			Route:   "workflows/{workflowName}",
+			Route:   "workflows/{workflowComponent}{workflowType}/{instanceId}",
 			Version: apiVersionV1alpha1,
 			Handler: a.onStartWorkflow,
 		},
 		{
 			Methods: []string{fasthttp.MethodDelete},
-			Route:   "workflows/{workflowId}/{instanceId}",
+			Route:   "workflows/{workflowComponent}{workflowType}/{instanceId}",
```

Comment:
Does there need to be a slash in between `{workflowComponent}` and `{workflowType}` here, and in the above routes?
```suggestion
			Route:   "workflows/{workflowComponent}/{workflowType}/{instanceId}",
```